src/main/java/uk/gov/digital/ho/hocs/workflow/api/WorkflowService.java

```diff
@@ -81,49 +81,29 @@ public WorkflowService(CaseworkClient caseworkClient,
 
     public CreateCaseResponse createCase(String caseDataType, LocalDate dateReceived, List<DocumentSummary> documents, UUID userUUID, UUID fromCaseUUID, Map<String, String> receivedData) {
         // Create a case in the casework service in order to get a reference back to display to the user.
+        Map<String, String> seedData = new HashMap<>(receivedData);
         CreateCaseworkCorrespondentRequest correspondentRequest = null;
 
-        Map<String, String> caseData = new HashMap<>();
-        caseData.put(WorkflowConstants.DATE_RECEIVED, dateReceived.toString());
-        caseData.put(WorkflowConstants.LAST_UPDATED_BY_USER, userUUID.toString());
-
-        if (receivedData != null) {
-            // If FOI and we have a name we have correspondent details attached to the case creation reques
-            if (Objects.equals(caseDataType, WorkflowConstants.CASE_DATA_TYPE_FOI)
-                    && receivedData.containsKey(WorkflowConstants.FULL_NAME)) {
-                correspondentRequest = buildCorrespondentRequest(receivedData);
-
-                caseData.put(WorkflowConstants.KIMU_DATE_RECEIVED, receivedData.get(WorkflowConstants.KIMU_DATE_RECEIVED));
-                caseData.put(WorkflowConstants.TOPICS, receivedData.get(WorkflowConstants.TOPICS));
-                caseData.put(WorkflowConstants.ORIGINAL_CHANNEL, receivedData.get(WorkflowConstants.ORIGINAL_CHANNEL));
-                caseData.put(WorkflowConstants.REQUEST_QUESTION, receivedData.get(WorkflowConstants.REQUEST_QUESTION));
-            } else if (isCreationForCompWebformCase(caseDataType, receivedData)) {
-                caseData.putAll(receivedData);
-            }
+        if (Objects.equals(caseDataType, WorkflowConstants.CASE_DATA_TYPE_FOI)
+                && receivedData.containsKey(WorkflowConstants.FULL_NAME)) {
+            seedData = new HashMap<>();
```

Comment:
Info: 
You could just have cleared the map - while this isn't an issue with this size of map, its always more performance to clear.